File: internal/content/testdata/fixtures/sample.go
// Package samplefixture is a test fixture for the source content-type
// family. The CLOC totals — line_count, loc, comment_loc, blank_loc —
// are exercised by TestFixturesAttributeSpotChecks.
package samplefixture

/*
Block comment spanning
three lines — counts as
3 comment_loc.
*/

import "fmt"

// Greet prints a greeting.
func Greet(name string) {
	fmt.Println("hello,", name)
}

File: internal/content/testdata/fixtures/sample.rs
// Sample Rust fixture for the source content-type family.
// Mirrors sample.go's structure for cross-language comparison.

/// Documentation comment — Rust uses `///` and `//!`. Both start
/// with `//` so our line-comment classifier handles them.

fn greet(name: &str) {
    println!("hello, {}", name);
}

fn main() {
    greet("world");
}
